src/main/scala/com/ironcorelabs/UserCreateOpts.scala

```diff
@@ -0,0 +1,15 @@
+package com.ironcorelabs.scala.sdk
+
+import cats.effect.Sync
+import com.ironcorelabs.{sdk => jsdk}
+
+case class UserCreateOpts(
```

Comment:
nitpick: this case class can be one line.

Reply:
Fixed